e2e_tests/docker_image/generate-keys.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 Contributors to the Parsec project.
+# SPDX-License-Identifier: Apache-2.0
+
+# These commands are made to create keys and key mappings using an old version of Parsec.
+# One test will try to use those keys again to make sure they still work.
+
+set -xeuf -o pipefail
+
+mkdir /tmp/create_keys
+
+# Use an old version of the Parsec service to make sure keys can still be used
+# with today's version.
+git clone https://github.com/parallaxsecond/parsec.git --branch 0.7.0 /tmp/create_keys/parsec
+cd /tmp/create_keys/parsec
+cargo build --features "all-providers, all-authenticators"
+
+# Start the service with all providers
+tpm_server &
+sleep 5
+tpm2_startup -c -T mssim
+sleep 2
+tpm2_changeauth -c owner tpm_pass -T mssim
+cd /tmp/create_keys/parsec/e2e_tests
+SLOT_NUMBER=`softhsm2-util --show-slots | head -n2 | tail -n1 | cut -d " " -f 2`
+find . -name "*toml" -not -name "Cargo.toml" -exec sed -i "s/^# slot_number.*$/slot_number = $SLOT_NUMBER/" {} \;
+cd ../
+./target/debug/parsec -c e2e_tests/provider_cfg/all/config.toml &
+
+# We use the Parsec Tool to create one RSA and one ECC key per provider,
+# when it is possible.
+cargo install parsec-tool
+export PARSEC_SERVICE_ENDPOINT="unix:/tmp/parsec.sock"
+/opt/rust/bin/parsec-tool -p 1 create-rsa-key -k rsa
```

Comment:
Just out of curiosity - isn't it possible to just do `parsec-tool ...`, do you need the full path? Maybe we can add that folder, `/opt/rust/bin` to `PATH` for the whole image?

Reply:
I don't mind either way - you still have to update the workflow file anyway, but I understand if you don't want to rebuild some of the image